### PR TITLE
Bluetooth: Mesh: Enabled CDP1 in NLC supported samples

### PR DESCRIPTION
**Pull request by PavelVPV**
NLC profiles implicitly enable Mesh Protocol 1.1 support. In Mesh Protocol v1.1 the Composition Data Page 1 shall be supported by the node. This option is not enabled by default, so we need to enable it explicitly.

### CODE REVIEW

**Review by Andrewpini — APPROVED** [review]

**Review by ludvigsj — APPROVED** [review]

File: subsys/bluetooth/mesh/Kconfig [hunk 1]
```diff
@@ -52,6 +52,7 @@ config BT_MESH_NLC_PERF_CONF
 	bool "Enable NLC performance configurations"
 	select BT_MESH_V1d1
 	select BT_MESH_COMP_PAGE_2
+	select BT_MESH_COMP_PAGE_1
```
Comment (alxelax) [inline subsys/bluetooth/mesh/Kconfig:55]:
```
warning: BT_MESH_COMP_PAGE_1 (defined at subsys/bluetooth/host/../mesh/Kconfig:1388, subsys/bluetooth/host/../mesh/Kconfig:1388) has direct dependencies (BT_MESH_MODEL_EXTENSIONS && BT_MESH_V1d1 && BT_MESH && BT_HCI_HOST && BT_RPC_STACK) || (BT_MESH_MODEL_EXTENSIONS && BT_MESH_V1d1 && BT_MESH && BT_HCI_HOST && BT_HCI && BT) with value n, but is currently being y-selected by the following symbols:
 - BT_MESH_NLC_PERF_CONF (defined at /home/alkh-local/ncs/nrf/subsys/bluetooth/mesh/Kconfig:51), with value y, direct dependencies BT_MESH && BT (value: y), and select condition BT_MESH && BT (value: y)
```
Seems it needs more depende

Comment (alxelax) [reply subsys/bluetooth/mesh/Kconfig:55]: light_dimmer sample. Didn't check the others.

Comment (PavelVPV) [reply subsys/bluetooth/mesh/Kconfig:55]: It gets enabled: `with value n, but is currently being y-selected`, but Ok, I will explicitly enable `BT_MESH_MODEL_EXTENSION`.

Comment (PavelVPV) [reply subsys/bluetooth/mesh/Kconfig:56]: Fixed

**Review by alxelax — APPROVED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -375,15 +375,15 @@ Bluetooth mesh samples
 
 * :ref:`bluetooth_mesh_light_lc` sample:
 
-  * Added Composition Data Page 2 support.
-    Composition Data Page 2 support has a dependency on Bluetooth mesh 1.1 support.
+  * Added Composition Data Pages 1 and 2 support.
+    Composition Data Pages 1 and 2 support has a dependency on Bluetooth mesh 1.1 support.
```
Comment (mia-ko) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:379]:
```suggestion
  * Added support for Composition Data Pages 1 and 2.
    Support for Composition Data Pages 1 and 2 has a dependency on Bluetooth mesh 1.1 support.
```

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -375,15 +375,15 @@ Bluetooth mesh samples
...
   * Fixed an issue where the sample could return an invalid Light Lightness Status message if the transition time was evaluated to zero.
   * Removed support for the configuration with :ref:`CMSE enabled <app_boards_spe_nspe_cpuapp_ns>` for :ref:`zephyr:thingy53_nrf5340`.
 
 * :ref:`bluetooth_mesh_light_dim` sample:
 
-  * Added Composition Data Page 2 support.
-    Composition Data Page 2 support has a dependency on Bluetooth mesh 1.1 support.
+  * Added Composition Data Pages 1 and 2 support.
```
Comment (mia-ko) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:385]: See previous comment

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -392,8 +392,8 @@ Bluetooth mesh samples
 
 * :ref:`bluetooth_mesh_sensor_server` sample:
 
-  * Added Composition Data Page 2 support.
-    Composition Data Page 2 support has a dependency on Bluetooth mesh 1.1 support.
+  * Added Composition Data Pages 1 and 2 support.
```
Comment (mia-ko) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:395]: The same comment as above

**Review by mia-ko — CHANGES_REQUESTED** [review]
Introducing support for two pages required some sentence work for better flow, I felt :)
See if you agree.

**Comment by PavelVPV** [comment]
I initially aligned the extensions list size to 2.5.0-rc1, but noticed that some fixes were done to CDP1 after the tag, so I adjusted the values according to the current main branch.

**Review by mia-ko — APPROVED** [review]

**Review by ludvigsj — APPROVED** [review]

**Review by alxelax — APPROVED** [review]

**Pull request merged**